### ISSUE TITLE
Hero Icons thumb icon renamed

### ISSUE THREAD

**Issue by jmcannon** [styled-icons/styled-icons#2120]
Just wasted an embarrassing amount of time before realizing that the thumb icon in hero icons has been renamed to HandThumbUp and HandThumbDown (from ThumbUp and ThumbDown). That change isn't reflected yet on the website.

**Comment by warrenmcquinn** [comment]
I've also encountered naming issues with the following heroicons since upgrading to 10.47.0:

- ChartSquareBar (now ChartBarSquare)
- Refresh (now ArrowPath)
- PencilAlt (now PencilSquare)
- Collection (now CircleStack)
- Search (now MagnifyingGlass)
- Template (now RectangleGroup)
- X (now XMark)
- LightningBolt (now Bolt)
- ViewGrid (now Squares2x2)
- ViewList (now Bars4)

**Comment by ChiefORZ** [comment]
yes, we encountered this too. such a api change/renaming is a breaking change in my books.

**Comment by ValentynaTemy** [comment]
Found the full list of renamings: https://github.com/tailwindlabs/heroicons/releases/tag/v2.0.0 
In addition, some icons have changed their appearance, I encountered changes in: **Reply** & **ClipboardCheck**.

It looks like a breaking change
